information-system/data-store/access-control.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: default
+title: Access Control
+nav_order: 9
+grand_parent: Information System
+parent: Data store
+---
+
+{: .no_toc }
+
+<details  open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+{: .text-delta }
+* TOC
+{:toc}
+____
+</details>
+
+# Access Control Overview
+
+This document will discuss the types of access controls available and then reveal the methods to apply these access controls through the Datastore User Interface. 
+
+## What access controls are available for datastore datasets?
+ 
+There are three access rights to consider: 
+
+1. Metadata access
+2. Data access
+3. Admin access
+
+
+## Metadata access
+
+Facilitates control of general viewing and metadata updating.
+
+### Metadata Read
+
+- If enabled, users with this right will be able view the dataset's metadata and find the dataset in the datastore "search" and "list" tools. 
+- If disabled, users will not be able to find the dataset in the datastore let alone view the metadata.
+
+
+### Metadata Write
+
```

Comment:
Would propose a "Automatically grants" section below each access level to more clearly describe the inherited access e.g. 

### Metadata Write

#### Also grants

- Metadata read

Reply:
Resolved